python/salpytools/salpylib.py

```diff
@@ -72,18 +75,18 @@ class DeviceState:
     HeaderService
     '''
 
-    def __init__(self, Device='atHeaderService',default_state='OFFLINE',
+    def __init__(self, Device='atHeaderService', default_state='OFFLINE',
```

Comment:
The naming conventions in this file seem to vary, with underscores, fully-capitalized words, initially-capitalized words, and camelCase used in various places.  I can see some of that being desirable for consistency with SAL, but some appears gratuitous.

You may want to start with a comment that mentions all the cases where this is for SAL consistency.

Reply:
While I agree that the naming convention should be more consistent, it not clear to what the long term status of this module would be. Moreover, changing the names of the variable across the library is not trivial. My understanding and expectation is that it should be become part of SAL in the future. If that's not the case it could make sense to do this. To avoid forgetting this issue I created a ticket to capture this request,  DM-15864.